curl/tests/client_test.cc

```diff
@@ -0,0 +1,227 @@
+#include <catch.hpp>
+#include <leatherman/curl/client.hpp>
+#include <leatherman/curl/request.hpp>
+#include <leatherman/curl/response.hpp>
+#include "mock_curl.hpp"
```

Comment:
AppVeyor's failing because `curl/curl.h` is included by the Leatherman headers. `mock_curl.hpp` should be included before them.